services/knative/0.18.3/knative-serving.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: knative
+  namespace: ${releaseNamespace}
+spec:
+  dependsOn:
+    - name: kube-prometheus-stack
+      namespace: ${releaseNamespace}
```

Comment:
I think I'm a bit late, but I think KNative can't be installed without Istio since it uses some of its CRDs. Prometheus is not required for KNative

Reply:
@alembiewski @farhan5900 Does this still need to be fixed?

Reply:
As for the Istio, please ignore my comment - it's been verified that KNative can be installed without one. 
But I think we should remove `kube-prometheus-stack` from the `dependsOn` list since I don't think it is the case.  
Also, I think some metadata (icon, overview) is still missing for this service.